From a48a3758d16c4e13249229a74338162025958fbd Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Tue, 18 Mar 2014 09:15:52 +0800
Subject: [PATCH 0773/1074] ENGR00303795-3 usb: chipidea: imx: rename host
 quirk flag to reflect its real meaning

This flag is not only used for mxs phy's bug, but also
for speical routine for other imx host operations.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 drivers/usb/chipidea/ci_hdrc_imx.c |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/usb/chipidea/ci_hdrc_imx.c b/drivers/usb/chipidea/ci_hdrc_imx.c
index e118714..65711b3 100644
--- a/drivers/usb/chipidea/ci_hdrc_imx.c
+++ b/drivers/usb/chipidea/ci_hdrc_imx.c
@@ -29,7 +29,7 @@
 
 #define CI_HDRC_IMX_IMX28_WRITE_FIX		BIT(0)
 #define CI_HDRC_IMX_SUPPORT_RUNTIME_PM		BIT(1)
-#define CI_HDRC_IMX_MXS_PHY_EHCI_QUIRK		BIT(2)
+#define CI_HDRC_IMX_HOST_QUIRK			BIT(2)
 #define CI_HDRC_IMX_HAS_HSIC			BIT(3)
 
 struct ci_hdrc_imx_platform_flag {
@@ -40,17 +40,17 @@ static const struct ci_hdrc_imx_platform_flag imx27_usb_data = {
 };
 
 static const struct ci_hdrc_imx_platform_flag imx23_usb_data = {
-	.flags = CI_HDRC_IMX_MXS_PHY_EHCI_QUIRK,
+	.flags = CI_HDRC_IMX_HOST_QUIRK,
 };
 
 static const struct ci_hdrc_imx_platform_flag imx28_usb_data = {
 	.flags = CI_HDRC_IMX_IMX28_WRITE_FIX |
-		CI_HDRC_IMX_MXS_PHY_EHCI_QUIRK,
+		CI_HDRC_IMX_HOST_QUIRK,
 };
 
 static const struct ci_hdrc_imx_platform_flag imx6q_usb_data = {
 	.flags = CI_HDRC_IMX_SUPPORT_RUNTIME_PM |
-		CI_HDRC_IMX_MXS_PHY_EHCI_QUIRK |
+		CI_HDRC_IMX_HOST_QUIRK |
 		CI_HDRC_IMX_HAS_HSIC,
 };
 
@@ -296,7 +296,7 @@ static int ci_hdrc_imx_probe(struct platform_device *pdev)
 		data->supports_runtime_pm = true;
 	}
 
-	if (imx_platform_flag->flags & CI_HDRC_IMX_MXS_PHY_EHCI_QUIRK)
+	if (imx_platform_flag->flags & CI_HDRC_IMX_HOST_QUIRK)
 		pdata.flags |= CI_HDRC_IMX_EHCI_QUIRK;
 
 	if (data->usbmisc_data && data->usbmisc_data->index > 1 &&
-- 
1.7.5.4

